### ISSUE TITLE
JSON Schema

### ISSUE THREAD

**Issue by josee-sabourin** [MobilityData/gbfs#272]
MobilityData has drafted a JSON schema for GBFS, it will have its own [repository](https://github.com/MobilityData/gbfs-json-schema) and will be managed by MobilityData, please have a look and let us know what you think either in this issue or by opening a PR/issue on the schema repo itself.

Some open questions: 
- The schemas are written to be a 1:1 representation of the versions of the spec, does this correspond with how you intend on using the schemas? (See the versions using tags, master branch corresponds to v2.1-RC2)
- Are descriptions for fields sufficient, should they mirror exactly what is written in the spec? At what point does the schema essentially just become a mirror image of the spec? Should only essential details be in the schemas? 
  - More on descriptions: in `system_pricing_plans.json` the `description` field causes a conflict given that `description` is a keyword in the schema vocabulary, should this field by changed to avoid mistakes? 
- We're looking for a way to reference the geojson schema within the larger `geofencing_zones.json` schema, perhaps someone with more experience in schemas will be able to help us with that!

**Comment by thekaveman** [comment]
Hello! Long time lurker on here, just wanted to drop a line with some questions/thoughts:

* Can you share some of the thinking behind putting the schemas in a separate repo? And/or in a separate organization? This feels... a little funny. In my mind, the schemas should live as close as possible to the spec (same repo, or at least same org).

* I've done a lot of work on the [MDS schemas](https://github.com/openmobilityfoundation/mobility-data-specification/tree/main/schema), and I'm happy to offer any assistance or advice that I can for this effort.

**Comment by LeoFrachet** [comment]
Oh, thanks Kegan for the question, it’s a really good one! (And thank you, too, for the offer of help. We’ll be in touch about that!)

GBFS was created in 2014 by Mitch Vars (@mplsmitch) — who is incidentally now working with us at MobilityData — and then NABSA’s endorsement, support, and hosting was key to its success starting in 2015. But, especially given the pace of change in the industry, NABSA lacked the internal resources to maintain the GBFS data format. To address that, first they put out the RFP in 2018 looking for support, and this is how MobilityData initially started working on GBFS under contract to NABSA. But after that contract was completed, we transitioned to [a partnership](https://www.linkedin.com/feed/update/urn:li:activity:6696135078369730561/). As a result, MobilityData has taken charge of GBFS governance, improvement, and extensions, and we have raised funds ourselves to support this work. 

But MobilityData envisions its role in the shared mobility data ecosystem as broader than only the specification. On the transit side, if you glance at what we are going with GTFS, you’ll see that the spec is still hosted by Google, its co-creator, but that MobilityData has now developed:
- A Canonical GTFS Validator, hosted on our GitHub repository: github.com/MobilityData/gtfs-validator;
- A Grading Scheme, which is in beta phase and will be published on our resource center;
- A Resource Center (gtfs.mobilitydata.org), containing the specification, the Best Practices around the specification, examples, how to guide and links to other resources. (We have started this for GBFS as well on gbfs.mobilitydata.org)

We are also working on a suite of other projects, which will eventually extend to GBFS as well:
- The Mobility Database (mobilitydatabase.org, still in staging phase for now), hosting semantical data on the available datasets, and the list of all GTFS datasets available (NABSA has historically shied away from active maintenance of the GBFS dataset catalog that is currently part of the GBFS repo);
- The Mobility Archives, archiving snapshots of datasets for eternity and beyond, for which the right hosting solution is currently being evaluated;
- OpenMobilityData (openmobilitydata.org), a UI allowing to search and discover those datasets.

We are also providing training and workshops worldwide, this year in three languages (English, French and Spanish), hopefully next year in four or five languages (adding German & Japanese).

The JSON Schema may be the tip of the iceberg that you are seeing from the MobilityData work, but our broader vision is the reason why, when the question of “on which repo should the JSON schema be hosted” landed on my desk a few days ago, I decided that it should be hosted in our organization.
